CRM/Utils/System/Standalone.php

```diff
@@ -205,7 +205,7 @@ public function url(
   ) {
     $fragment = $fragment ? ('#' . $fragment) : '';
     if ($absolute) {
-      return Civi::paths()->getUrl("[cms.root]/{$path}?{$query}$fragment");
+      return Civi::paths()->getUrl("[cms.root]/{$path}?{$query}$fragment", 'absolute');
     }
     else {
       return "/{$path}?{$query}$fragment";
```

Comment:
Looks like this code should be what the above used to be (generating a relative path):

```suggestion
      return Civi::paths()->getUrl("[cms.root]/{$path}?{$query}$fragment");
```